docs/conf/modules.conf.example

```diff
@@ -2048,7 +2048,7 @@
 
 # You must define <sasl:target> to the name of your services server so
 # that InspIRCd knows where to send SASL authentication messages and
-# when it should enable the SASL capability.
+# when it should enable the SASL capability.
```

Comment:
Can you revert this and the added blank line at the end of the file?

Reply:
I apologize on behalf of my silly editor!